_includes/components/edit-buttons.html

```diff
@@ -0,0 +1,13 @@
+{% assign indicator_id_slug = meta.indicator | slugify %}
+
+{% if site.data_edit_url %}
+<a class="btn btn-primary" href="{{ site.data_edit_url | replace: '[id]', indicator_id_slug }}">{{ t.indicator.edit_data }}</a>
+{% else %}
+<a class="btn btn-primary" href="http://prose.io/#{{ site.org_name }}/{{ site.repo_name }}/edit/{{ site.branch }}/{{ site.data_dir_for_edits | default: 'data' }}/indicator_{{ meta.indicator | slugify }}.csv">{{ t.indicator.edit_data }}</a>
```

Comment:
Although I understand the reasoning behind this, I find it rather strange that although you have a dedicated `yaml` file for URLs like this, you still need to hardcode a default.

Reply:
@SavvasStephanides Yeah I agree it would be better not to have the default at all. I just included it for backwards compatibility. In general I've found it easier to code in backwards compatibility when possible. This allows us to quietly change the documentation - meaning new users will use the new config, while ensuring that existing users' sites will not break.